src/main/views/error.njk

```diff
@@ -1,8 +1,5 @@
-<html>
-  <head>
-  </head>
-  <body>
-    <h1>Something went wrong</h1>
-    <p>TODO: replace with a proper template in your application</p>
-  </body>
-</html>
+{% extends "./layouts/default.layout.njk" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">Something went wrong</h1>
```

Comment:
Do you want to start using translation.json, the file i showed you last week for welsh translation?

Reply:
We should do this at some point for sure

Reply:
We can do this in static files for now. But for data being fed via the server should we not at least wait until our model is defined before we do so?